common/views/components/PageWrapper/PageWrapper.js

```diff
@@ -104,19 +102,20 @@ const PageWrapper = (Comp: NextComponent) => {
         text: asHtml(globalAlertData.text),
         isShown: globalAlertData.isShown === 'show'
       };
+
       // There's a lot of double checking here, which makes me think we've got
       // the typing wrong.
-      const openingTimes = context.req
-        ? await fetchOpeningTimes(context.req)
-        : clientStore && clientStore.get('openingTimes');
-
       const toggles = context.req
         ? context.query.toggles
         : clientStore && clientStore.get('toggles');
 
+      const openingTimes = context.req
+        ? parseOpeningHours(parseVenuesToOpeningHours(context.query.openingTimes))
+        : clientStore && clientStore.get('openingTimes');
+
       if (serverStore) {
-        serverStore.set('openingTimes', openingTimes);
         serverStore.set('toggles', toggles);
+        serverStore.set('openingTimes', openingTimes);
```

Comment:
Instead of using this pageStore, which I think we should deprecate (it's not available to other `getInitialProps`), could we send it down the [`ExtraProps`](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/views/components/PageWrapper/PageWrapper.js#L95-L97)?